rdkit/Chem/SaltRemover.py

```diff
@@ -146,6 +196,13 @@ def StripMol(self, mol, dontRemoveEverything=False):
     2
 
     """
+    m, removed = self._StripMol(mol, dontRemoveEverything)
+    return m
+
+  def StripMolAndReturnRemoved(self, mol, dontRemoveEverything=False):
+    return self._StripMol(mol, dontRemoveEverything)
```

Comment:
Making the intention clearer is always a good think. And if you did not understand the change at a first glance, then the change makes definitely sense.
Ideally, if this variable is not needed, then we should actually get rid of it. Is there a way in **Python** to partly denormalize a _tuple_?